### ISSUE TITLE
Death Star bug

### ISSUE THREAD

**Issue by tonyviroos** [steemnova/steemnova#228]
If the defender or attacker has tech below 10 everything is fine, but if both the defender and attacker have tech above 10 then death stars become indestructible.

Combat simulation when attacker has tech at 10 and defender at 9, then it's ok.
![Screenshot_2020-10-26 Simulator - Star Platinum - SteemNova(1)](https://user-images.githubusercontent.com/6677107/97237348-32142f80-17e7-11eb-9513-fa441906371c.png)
![Screenshot_2020-10-26 Fight report - Star Platinum - SteemNova(1)](https://user-images.githubusercontent.com/6677107/97237355-35a7b680-17e7-11eb-80eb-639ba89508a9.png)

Combat simulation when the attacker and defender have tech at 10, the death stars are not destroyed when they should.
![Screenshot_2020-10-26 Simulator - Star Platinum - SteemNova](https://user-images.githubusercontent.com/6677107/97237472-7a335200-17e7-11eb-9e91-7106935cbaea.png)

![Screenshot_2020-10-26 Fight report - Star Platinum - SteemNova](https://user-images.githubusercontent.com/6677107/97237484-828b8d00-17e7-11eb-9009-628bf1dc73f3.png)

**Comment by IntinteDAO** [comment]
Of course, if you are using OPBE, you can expect various strange results, but according to you:
- SteemNova Battle Engine
- Ogame simulators

This result is correct. 
Unless I misunderstand something. In the sense of even 5 million, the Cruiser will not beat the 3 Death Stars

**Comment by tonyviroos** [comment]
This error also occurs on ttps: //steemnova.intinte.org/
